Assets/Scripts/Game Controllers/LevelEndTriggerController.cs

```diff
@@ -10,13 +10,22 @@ public class LevelEndTriggerController : MonoBehaviour
 
     private void Start()
     {
-        indexForNextScene = SceneManager.GetActiveScene().buildIndex + 1;
+        if (SceneManager.GetActiveScene().buildIndex == 0)
+        {
+            indexForNextScene = lastLevel + 1;
```

Comment:
I don't think this logic is needed in the long term because an infinite level wouldn't have a Level End Trigger. I think there should at least be a comment with that information here. Making the infinite level end without a Level End Trigger is getting into infinite level implementation, so I don't think it needs to be changed in this pull request.